app/assets/stylesheets/v3/layout_components/sections.scss

```diff
@@ -1,71 +1,65 @@
 .heading {
-  padding: $base-padding*0.75 $base-padding;
+  padding: $base-padding $base-padding $base-padding*2;
   color: $white;
 }
 
 .section {
-  margin-bottom: $base-margin;
+  margin-bottom: $base-margin*3;
 
   &__heading {
     @extend .heading;
-    background: $brand-blue-light;
+    background: transparent;
```

Comment:
I think we want this background to be white instead of transparent, because when I open the modal to edit VRAR the heading has no background.

Reply:
I thought about that, but I actually think it makes sense to leave it transparent (for example if we have sections with grey backgrounds) and then for things like the modals, we can set those to explicitly have a white background.